src/main/java/rest/CountryResource.java

```diff
@@ -11,13 +11,44 @@
 import errorhandling.APIUtilException;
 import errorhandling.NotFoundException;
 import facades.CountryFacade;
+import io.swagger.v3.oas.annotations.OpenAPIDefinition;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.info.Contact;
+import io.swagger.v3.oas.annotations.info.Info;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.servers.Server;
+import io.swagger.v3.oas.annotations.tags.Tag;
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.QueryParam;
 import javax.ws.rs.core.MediaType;
 
+@OpenAPIDefinition(
+        info = @Info(
+                title = "Hold Krykke Semesterprojekt API",
+                version = "1.0",
+                description = "API related to Cphbusiness 3rd semester CS project.",
+                contact = @Contact(name = "Github Contributors", url = "https://github.com/Hold-Krykke/3SemProject_BackEnd#contributors")
+        ),
+        tags = {
+            @Tag(name = "Events", description = "API endpoint for Events")
+        },
+        servers = {
+            @Server(
+                    description = "For Local host testing",
+                    url = "http://localhost:8080/3SEMPROJECT/api/resource/events?"
+            ),
+            @Server(
+                    description = "Server API",
+                    url = "https://runivn.dk/3SEMPROJECT/api/resource/events?"
```

Comment:
Note certain if these URLs are right.

Reply:
I believe it should simply be the base URL, Swagger does the rest:

`http://localhost:8080/3SEM/` (Local)
&
`https://runivn.dk/3SEMPROJECT/` (Remote)